python/ray/tests/test_metrics.py

```diff
@@ -550,30 +550,30 @@ class Actor:
     # These tests should be retried because it takes at least one second
     # to get the fresh new memory table. It is because memory table is updated
     # Whenever raylet and node info is renewed which takes 1 second.
-    assert (wait_for_condition(
+    (wait_for_condition(
         test_local_reference, timeout=30000, retry_interval_ms=1000) is True)
 
-    assert (wait_for_condition(
-        test_object_pineed_in_memory, timeout=30000, retry_interval_ms=1000) is
-            True)
+    (wait_for_condition(
```

Comment:
The `is True` and extra parenthesis should be removed now that `wait_for_condition` doesn't return a boolean any more (same for the invocations below)